command/tls_ca_create.go

```diff
@@ -112,6 +169,34 @@ func (c *TLSCACreateCommand) Run(args []string) int {
 		c.Ui.Error(commandErrorText(c))
 		return 1
 	}
+	if c.IsEmpty() {
+		c.domain = "nomad"
+	} else if c.IsEmpty() && c.days != 0 {
+		c.domain = "nomad"
+	} else {
+		if c.commonName == "" {
+			c.Ui.Error("please provide the -common-name flag when customizing the CA")
+			c.Ui.Error(commandErrorText(c))
+			return 1
+		}
+		if c.country == "" {
+			c.Ui.Error("please provide the -country flag when customizing the CA")
+			c.Ui.Error(commandErrorText(c))
+			return 1
+		}
+
+		if c.organization == "" {
+			c.Ui.Error("please provide the -organization flag when customizing the CA")
+			c.Ui.Error(commandErrorText(c))
+			return 1
+		}
+
+		if c.organizationalUnit == "" {
+			c.Ui.Error("please provide the -organizational-unit flag when customizing the CA")
```

Comment:
These errors are directly printed to the users console, so we should title the sentence.